wind.go

```diff
@@ -38,13 +38,13 @@ type Window struct {
 	eventx *Xfid
 	events []byte
 
-	owner       int
+	owner       int // TODO(fhs): change type to rune
```

Comment:
Why? It would appear to have only two values: 0 and 'E`. Maybe it should be a boolean? Regardless, not an issue for this CL.

Reply:
It's actually more than two values (just grep for `w\.Lock`). Sometimes it's 'M' (mouse), 'K' (keyboard), or the first character of the message written to the event file. Anyway, it just felt a bit strange to have to cast the value when using type inference:
```
c := 'F'
if ... { c = 'E' }
w.Lock(int(c))
```

Reply:
conceivably at some point, it could be a string constant?

Reply:
String constant also works for me, as long as it gets rid of some type casts.